site/content/en/docs/reference/kueue.v1beta1.md

```
@@ -1237,6 +1246,39 @@ This is recorded only when Fair Sharing is enabled in the Kueue configuration.</
 </tbody>
 </table>
 
+## `ClusterQueueWorkloadDefaults`     {#kueue-x-k8s-io-v1beta1-ClusterQueueWorkloadDefaults}
+    
+
+**Appears in:**
+
+- [ClusterQueueSpec](#kueue-x-k8s-io-v1beta1-ClusterQueueSpec)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)